### ISSUE TITLE
1.5.1-beta.3 cannot upgrade

### ISSUE THREAD

**Issue by brucejo75** [meteor/meteor#8830]
I get a package incompatibility error.  Upgrade to 1.5.1-beta.2 works fine.

```
meteor update --release 1.5.1-beta.3
=> Errors while initializing project:
While selecting package versions:
error: Conflict: Constraint ddp-client@1.2.5 is not satisfied by ddp-client 2.0.0-beta.3.
Constraints on package "ddp-client":
* ddp-client@2.0.0-beta.3 <- top level
* ddp-client@1.2.5 <- ddp 1.2.5 <- accounts-base 1.3.1-beta.3
* ddp-client@2.0.0-beta.3 <- ddp-server 2.0.0-beta.3 <- ddp 1.2.5 <- accounts-base 1.3.1-beta.3

Conflict: Constraint ddp-server@1.2.6 is not satisfied by ddp-server 2.0.0-beta.3.
Constraints on package "ddp-server":
* ddp-server@2.0.0-beta.3 <- top level
* ddp-server@1.2.6 <- ddp 1.2.5 <- accounts-base 1.3.1-beta.3
```

**Comment by abernix** [comment]
Thanks for pointing this out, @brucejo75.  I think the `ddp` package should have got a bump to `-beta.3` in https://github.com/meteor/meteor/commit/7cb7866b6096c4cca44d38612d69a5fc07a215f9 as well, much like it did [on `release-1.6` for `-beta.2`](https://github.com/meteor/meteor/blob/release-1.6/packages/ddp/package.js).

**Comment by benjamn** [comment]
This should be fixed if you run `meteor update --release 1.5.1-beta.4`. Please feel free to reopen if the same problem persists, and thanks for reporting this issue @brucejo75!
